src/main/sh/buildApp.sh

```diff
@@ -0,0 +1,11 @@
+#!bin/bash
+cd /home/$USERNAME/bookingapplication/Booking_app
```

Comment:
To można by zrobić wykorzystując bieżące katalogi, zamiast wymuszać tworzenie dodatkowych w systemie :)

Reply:
Odpowiedź taka jak u góry. Po przeanalizowaniu scriptu stwiedzam, że niepotrzebnie chciałem wymusić tworzenie katalogów i przechodzenie między nimi. Script znajduje się w projekcie, więc jest to niepotrzebne.